### PR TITLE
feat: streaming prediction

### PR DESCRIPTION
**Pull request by andrewferlitsch**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
Add streaming prediction notebook
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/prediction/llm_streaming_prediction.ipynb [hunk 1]
```diff
@@ -0,0 +1,791 @@
+{
```
Comment (gericdong) [inline notebooks/official/prediction/llm_streaming_prediction.ipynb:1]: <p>Update the notebook links.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (gericdong) [inline notebooks/official/prediction/llm_streaming_prediction.ipynb:1]: <p>Incomplete sentence "Learn more about ."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (gericdong) [inline notebooks/official/prediction/llm_streaming_prediction.ipynb:1]: <p>It'd be better to use the official names of the models. See https://cloud.google.com/vertex-ai/docs/generative-ai/learn/models</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (gericdong) [inline notebooks/official/prediction/llm_streaming_prediction.ipynb:1]: <ul><li>Why the models are not consistent between these two places, one is using "google/text-bison@001" and another is using "text-bison".</li><li>No need to import again "from vertexai.preview.language_models import TextGenerationModel"</li></ul><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (gericdong) [inline notebooks/official/prediction/llm_streaming_prediction.ipynb:1]: <p>ext &gt; Next?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (gericdong) [inline notebooks/official/prediction/llm_streaming_prediction.ipynb:1]: <p>No need to import the modules again "from vertexai.preview.language_models import TextGenerationModel</p><p>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (gericdong) [inline notebooks/official/prediction/llm_streaming_prediction.ipynb:1]: <p>pandas &gt; Pandas</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Comment (gericdong) [inline notebooks/official/prediction/llm_streaming_prediction.ipynb:1]: <p>Do we need to clean up the tuning job?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (gericdong) [inline notebooks/official/prediction/llm_streaming_prediction.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #7.    if delete_bucket or os.getenv("ID_TESTING"):
</pre><p>ID_TESTING &gt; IS_TESTING</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (andrewferlitsch) [reply notebooks/official/prediction/llm_streaming_prediction.ipynb:1]: pandas is not capitalized: https://pandas.pydata.org/docs/user_guide/10min.html

Comment (andrewferlitsch) [reply notebooks/official/prediction/llm_streaming_prediction.ipynb:1]: done

Comment (andrewferlitsch) [reply notebooks/official/prediction/llm_streaming_prediction.ipynb:1]: done

Comment (andrewferlitsch) [reply notebooks/official/prediction/llm_streaming_prediction.ipynb:1]: done

Comment (andrewferlitsch) [reply notebooks/official/prediction/llm_streaming_prediction.ipynb:1]: done

Comment (andrewferlitsch) [reply notebooks/official/prediction/llm_streaming_prediction.ipynb:1]: TBD once Product Team updates me

Comment (andrewferlitsch) [reply notebooks/official/prediction/llm_streaming_prediction.ipynb:1]: done

**Review by gericdong — APPROVED** [review]

**Pull request merged**